runtime/isolated-vm/package.json

```diff
@@ -54,5 +54,5 @@
     "test": "node test.js || nodejs test.js"
   },
   "types": "isolated-vm.d.ts",
-  "version": "4.4.1"
+  "version": "4.4.2"
```

Comment:
I didn't exactly intend to update this, it's just what happened when I ran `yarn add isolated-vm`

Reply:
can you make bs locally? 

this PR doesn't change yarn.lock and isolated-vm is still 4.4.1 in yarn.lock

```
isolated-vm@^4.4.1:
  version "4.4.1"
  resolved "https://registry.yarnpkg.com/isolated-vm/-/isolated-vm-4.4.1.tgz#651ce31e435e769b2a356bb03db2553a478acebe"
  integrity sha512-5aDwxQGm78vHS+qJeUli2ILroG7OS/k3D/Mc0kcT9vyujiL4bV7PYYix1mAvuBm3v44nz2qcfAOqgAbhuACc/w==
```

Reply:
`make bs` had been succeeding locally, FWIW, but reverting the isolated-vm changes worked, thank you!